src/frontend/.eslintrc-root.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
   },
   overrides: [
     {
-      files: ['*.spec.*', '*.test.*'],
```

Comment:
The title for commit `07d877b23c5657d12970c78131fdfea2e08180d9` does not provide enough information